core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/repair/TestOnDemandRepairSchedulerImpl.java

```diff
@@ -129,6 +133,41 @@ public void testRestartRepairOnTable() throws EcChronosException
 
         OnDemandRepairSchedulerImpl repairScheduler = defaultOnDemandRepairSchedulerImplBuilder().build();
 
+        try
+        {
+            Thread.sleep(1000);
+        }
+        catch (InterruptedException e)
+        {
+        }
+
+        verify(scheduleManager).schedule(any(ScheduledJob.class));
```

Comment:
Rather than the Thread.sleep() above we could change verify to:

`verify(scheduleManager, timeout(1000)).schedule(any(ScheduledJob.class));`

On the first verification.